app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -29,6 +45,17 @@ define(function (require, exports, module) {
 
     openPanel: function () {
       this.$('.settings-unit').addClass('open');
+      var $input = this.$('.open input:first-of-type');
```

Comment:
What if we just used the 'autofocus' attribute on the inputs or buttons? The `autofocus` attribute is the standard way to indicate an element should be focused, and I'd rather update the templates than make assumptions like this code makes. We also have to take care to *not* do this on [mobile devices](https://github.com/mozilla/fxa-content-server/blob/25cf2761b0f274ee22c522f5e7361604a52cfe47/app/scripts/views/base.js#L393:L396).

Perhaps the autofocus code in base.js can be extracted into a function and used here.